src/gui/guiFormSpecMenu.cpp

```diff
@@ -2768,6 +2775,8 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 	m_dropdowns.clear();
 	theme_by_name.clear();
 	theme_by_type.clear();
+	field_close_on_enter.clear();
+	m_dropdown_index_event.clear();
```

Comment:
Please also add this to the destructor.

Reply:
Are you sure we need to?

Reply:
The other lists aren't cleared in the destructor; elements are only dropped and deleted.  I don't see how it really helps clarity or anything...